napari/layers/labels/labels.py

```diff
@@ -61,13 +113,22 @@ def __init__(
             selected_label=Event,
         )
 
-        self.seed = 0.5
-        self._image = label_image
-        self._image_view = None
-        self._meta = meta
-        self.interpolation = 'nearest'
-        self.colormap_name = 'random'
-        self.colormap = colormaps.label_colormap(num_colors)
+        self._data = labels
+        self._data_view = np.zeros((1, 1))
+        if metadata is None:
+            self.metadata = {}
+        else:
+            self.metadata = metadata
```

Comment:
As above, `self.metadata = metadata or {}`

Reply:
Note: the above expression is somewhat controversial, but I really like it. Happy to be overruled if it is deemed too arcane.

Reply:
it's fine for me, i like that it is one line. fixed